public/js/tests/tests.js

```diff
@@ -1,90 +1,79 @@
-test("should sort channels by name", function(assert) {
-  var channels = [
-    {friendlyName: "BBB"},
-    {friendlyName: "BBA"}
-  ];
+document.body.innerHTML = window.__html__['views/index.jade'];
 
-  var result = twiliochat.sortChannelsByName(channels)
+describe('TwilioChat', function() {
+  it('should sort channels by name', function() {
```

Comment:
As a recomendation we'd like to write the description of each test like this:

```javascript
it("sorts channels by name", function() {
  // ...
```

The _should_ word is implicit in the context of a test.

When you read the test's output you'll find it more understandable IMO, what do you think?